docs/xdebug.md

```diff
@@ -32,7 +32,7 @@ As example, an override could look like this:
 version: "3.4"
 
 services:
-  php:
+  php-fpm:
```

Comment:
The service name is not correct like this. See : https://github.com/dunglas/symfony-docker/blob/main/docker-compose.yml#L4

Reply:
Sorry my bad, I'm using my own docker-compose.yml thats the reason. I will change it.